.github/workflows/run-go-tests.yml

```
@@ -31,40 +31,34 @@ jobs:
-          sudo apt-get update -y
-          sudo apt-get install -y docker-compose
-          docker-compose version
+        uses: actions/checkout@v4
 
       - name: Start PostgreSQL and Neo4j Containers
         run: |
-          BH_POSTGRES_CONFIG=$(pwd)/.github/config/postgresql.conf docker-compose -f docker-compose.testing.yml up -d
+          BH_POSTGRES_CONFIG=$(pwd)/.github/config/postgresql.conf docker compose -f docker-compose.testing.yml up -d
 
       - name: Install Go
         uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)